client/lib/idds/client/clientmanager.py

```diff
@@ -119,6 +122,17 @@ def get_config_value(self, configuration, section, name, current, default):
         value = get_local_config_value(configuration, section, name, current, default)
         return value
 
+    def get_section(self, name):
+        name_sections = {'config': 'common',
+                         'auth_type': 'common',
+                         'host': 'rest',
+                         'x509_proxy': 'common',
```

Comment:
To make sure. Is it true that you want to move x509_proxy, oidc_token, and vo to the common section from separate sections? Also, does this require manual changes in existing idds_local.cfg?

Reply:
> Does panda have a idds_local.cfg file?

Yes.